.github/workflows/release.yml

```diff
@@ -212,6 +212,17 @@ jobs:
       run: |
         export TAG="$($HOME/.linkerd version --client --short)"
         go test -cover -race -v -mod=readonly ./cni-plugin/test -integration-tests
+  website_publish:
+    name: Linkerd website publish
```

Comment:
Can you please add a `if: startsWith(github.ref, 'refs/tags/stable') || startsWith(github.ref, 'refs/tags/edge')` here?

Reply:
Added 👍